lib/src/serializable.dart

```diff
@@ -159,16 +163,16 @@ class SerializableBus {
       return;
     }
 
+    List reflectedData = new List(data.length);
```

Comment:
Can we do better on this generic type?

Reply:
List<dynamic> ?

Reply:
Hmm, I guess there aren't really types on the items that get put into the list, so that's fine.